File: app/src/main/java/com/stone/lfernandosantos/storewars/models/Product.java
package com.stone.lfernandosantos.storewars.models;

import java.io.Serializable;

/**
 * Created by lf.fernandodossantos on 20/05/17.
 */

public class Product implements Serializable{
    public Long id;
    public String title;
    public Double price;
    public String zipcode;
    public String seller;
    public String thumbnailHd;
    public String date;
    public String carrinho;
    public String compra;

    public String getPrice() {

        Double p = price / 100;

        String priceFormat = String.format("%.2f",p);
        return priceFormat;
    }

    @Override
    public String toString() {
        return "Product{" +
                "title='" + title + '\'' +
                ", price=" + price +
                ", zipcode='" + zipcode + '\'' +
                ", seller='" + seller + '\'' +
                ", thumbnailHd='" + thumbnailHd + '\'' +
                ", date='" + date + '\'' +
                '}';
    }
}

File: app/src/main/java/douglasspgyn/com/github/desafiostone/business/model/Product.kt
package douglasspgyn.com.github.desafiostone.business.model

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import java.io.Serializable

/**
 * Created by Douglas on 12/11/17.
 */

@Entity(tableName = "products")
data class Product(@PrimaryKey var title: String = "",
                   var price: Double = 0.0,
                   var zipcode: String = "",
                   var seller: String = "",
                   var thumbnailHd: String = "",
                   var date: String = "",
                   var quantity: Int = 1) : Serializable